changelog/2.104.1.dd

```diff
@@ -0,0 +1,58 @@
+Ddoc
+
+$(CHANGELOG_NAV_INJECT)
+
+$(VERSION Jul 01, 2023, =================================================,
+
+$(CHANGELOG_HEADER_STATISTICS
+$(VER) comes with  11 fixed Bugzilla issues.
+        A huge thanks goes to the
+        $(LINK2 #contributors, 10 contributors)
+        who made $(VER) possible.)
+
+$(CHANGELOG_SEP_HEADER_TEXT_NONEMPTY)
+
+$(CHANGELOG_SEP_HEADER_TEXT)
+
+$(CHANGELOG_SEP_TEXT_BUGZILLA)
+
+$(BUGSTITLE_BUGZILLA DMD Compiler regression fixes,
+
+$(LI $(BUGZILLA 23965): [REG2.101.0] Appending deprecated structs in deprecated function causes deprecation message)
+$(LI $(BUGZILLA 23978): [REG 2.103.0] ICE: dip1021 memory corruption)
+$(LI $(BUGZILLA 23979): ICE on failed alias this attempt on pointer expression)
+$(LI $(BUGZILLA 24013): [REG 2.103.0] address of a  __traits$(LPAREN)getOverloads$(RPAREN) item is not converted to a delegate anymore)
+)
+$(BUGSTITLE_BUGZILLA DMD Compiler bug fixes,
+
+$(LI $(BUGZILLA 21025): Segfault for function contract -preview=dip1021)
+$(LI $(BUGZILLA 22729): ICE: Invalid array access for invalid interface definition)
+$(LI $(BUGZILLA 23768): ImportC: ICE on nested C initializer)
+$(LI $(BUGZILLA 23982): segfault when printing scope inference failure)
+$(LI $(BUGZILLA 23986): ICE: dip1021 asserts on `typeof$(LPAREN)null$(RPAREN)` parameter)
+)
+$(BUGSTITLE_BUGZILLA Phobos regression fixes,
+
+$(LI $(BUGZILLA 23976): std.range.slide fails in dmd-2.104.0)
+$(LI $(BUGZILLA 23993): std.algorithm.maxElement no longer works with array of BigInt)
+)
+)
+$(D_CONTRIBUTORS_HEADER 10)
+$(D_CONTRIBUTORS
+    $(D_CONTRIBUTOR Dennis)
+    $(D_CONTRIBUTOR Dennis Korpel)
```

Comment:
I'm in here twice?

Reply:
Did you change your git settings recently? Or maybe there's a discrepancy between github and your gitconfig (if you've made changes via github's ui)?

Reply:
This commit has you with a different name and address compared to the others.

https://github.com/dlang/dmd/commit/eeaeb89ba5789e2eabfdbd30e6a603f8beb12849